.idea/misc.xml

```diff
@@ -1,4 +1,4 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
-  <component name="ProjectRootManager" version="2" project-jdk-name="Python 3.7 (rpxdock)" project-jdk-type="Python SDK" />
+  <component name="ProjectRootManager" version="2" project-jdk-name="Python 3.6 (rpxdock) (2)" project-jdk-type="Python SDK" />
```

Comment:
I may have missed this in the last dicussions, is there a particular reason for why changing the python version?

Reply:
Hmm, I don't think this should have changed. I'm not sure why that actually changed.